contracts/DefifaGovernor.sol

```diff
@@ -109,6 +95,17 @@ contract DefifaGovernor is Governor, GovernorCountingSimple, IDefifaGovernor {
     return propose(_targets, _values, _calldatas, '');
   }
 
+  /**
+    @notice
+    Attests to a scorecard.
+
+    @param _scorecardId The scorecard ID. 
+  */
+  function attestToScorecard(uint256 _scorecardId) external override {
+    // Vote.
+    super._castVote(_scorecardId, msg.sender, 1, '');
```

Comment:
Nit: Having the UI call `castVoteWithReasonAndParams` (with the ordered tier ids of the user) is cheaper than calling `castVote`, could be a nice gas savings.

Reply:
maybe we have a 2nd option as `attestToScorecardWithSig` ?